### DISCUSSION TITLE
Hide Connection Manager Window (hide cm)

### DISCUSSION THREAD

**Discussion by fernandeusto** [rustdesk/rustdesk#6008] [Q&A] [answered]
Please reconsider the option of being able to run the client with the "hide connection manager window" option, otherwise I can only stop using Rustdesk to manage my company's computers.

![image](https://github.com/rustdesk/rustdesk/assets/87156183/d6920796-79c1-4084-bba0-61ea0c0fd859)

**Comment by rustdesk** [comment, marked as answer]
Removed because of [Kaspersky compliance](https://github.com/rustdesk/rustdesk/discussions/5624)

But you can enable it via strategy of `Pro` on web console.

![image](https://github.com/user-attachments/assets/bd991dbb-6be3-42f5-8bf1-3601a36a01be)

**Reply by rustdesk** [reply]
To make this work.
1. You need to make sure the strategy is assigned to the expected devices, and the strategy is not disabled.
2. You need to make sure the api works on your devices, the simplest way is to try to login with the client or check if you can see version (in `info` column) or connection logs to these devices on web console.

Below two links will help you when the api does not work
https://github.com/rustdesk/rustdesk/wiki/FAQ#failed-to-log-in-with-custom-client-build-or-public-release-with-wrong-api-settings
https://github.com/rustdesk/rustdesk/wiki/FAQ#connection-log-or-device-name-on-devices-page-of-web-console-is-empty

<img width="2328" height="296" alt="3da0cd16-5a3f-4788-b3af-7870012cee6c" src="https://github.com/user-attachments/assets/d87b38cc-f699-4916-bb9b-6148f7c80791" />


![image](https://github.com/user-attachments/assets/0f6a679c-8b1f-4847-a2bc-166ef7fc413e)


![image](https://github.com/user-attachments/assets/7fc43e2e-fbbf-44e1-864a-a80cc271a72f)

**Reply by rustdesk** [reply]
If you found this does not work for you. Please check below.

1. Make sure API works, https://github.com/rustdesk/rustdesk/wiki/FAQ#connection-log-or-device-name-on-devices-page-of-web-console-is-empty
2. Make sure you have set the password correctly. 
> One user set password with [`default-connect-password`](https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/#default-connect-password), this is not right, you should set password follow https://github.com/rustdesk/rustdesk/wiki/FAQ#how-can-we-set-up-a-client-with-a-fixed-password-for-unattended-remote-access
